.github/workflows/integrationtest-tool.yml

```diff
@@ -9,30 +9,29 @@ on:
       REPOSITORY_NAME:
         required: true
         type: string
-      MORYX_OPTIMIZE_CODE:
-        required: false
-        type: string
-      MORYX_BUILD_CONFIG:
-        required: false
-        type: string
-      MORYX_BUILDNUMBER:
-        required: false
-        type: string
-      DOTNET_SKIP_FIRST_TIME_EXPERIENCE:
-        required: false
-        type: boolean
 
 jobs:
   IntegrationTests:
-    runs-on: windows-latest
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      
+      # - name: Download build artifacts
+      #   uses: actions/download-artifact@v2
+      #   with:
+      #     name: ${{inputs.REPOSITORY_NAME}}-build
+      #     path: /home/runner/work/_temp/artifacts/
 
-      - name: Setup .NET SDK
+      - name: Setup .NET SDK standard (7)
         uses: actions/setup-dotnet@v1
         with:
           dotnet-version: ${{ inputs.dotnet_sdk_version }}
 
+      - name: Setup .NET SDK 6 for testing purposes
```

Comment:
Remember to remove

Reply:
This is not for our pipeline testing, but for unit- and integration tests, that are currently only working with .net 6. Therefore this is still necessary.

Reply:
Ok, weird but good to know 🤔 Please create an issue for that then in the repository so that we remember to investigate. If you have them available you could also help with the investigation when you provide errors logs you gathered before you found out that only .net 6 works atm 😊